modules/web/src/app/shared/components/addon-list/install-addon-dialog/component.ts

```diff
@@ -80,6 +82,11 @@ export class InstallAddonDialogComponent {
     return _.escape(getAddonShortDescription(this.addonConfigs.get(name)));
   }
 
+  getAddonDescription(name: string): SafeHtml {
```

Comment:
isn't `this._sanitizer.sanitize(SecurityContext.HTML, description) || '';` return a string ?? right ?
```suggestion
  getAddonDescription(name: string): string {
```